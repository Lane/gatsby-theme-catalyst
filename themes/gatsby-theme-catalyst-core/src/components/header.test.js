import React from "react"
import renderer from "react-test-renderer"
import SiteHeader from "./header"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SiteHeader />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})