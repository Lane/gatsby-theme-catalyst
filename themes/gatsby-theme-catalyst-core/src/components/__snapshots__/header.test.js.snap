// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="css-92rb9y-SiteHeader"
  id="header"
>
  <p>
    Header area for latent shadowing in sibling themes
  </p>
</header>
`;
